persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/configuration/TableManipulationConfigurationBuilder.java

```diff
@@ -155,7 +156,7 @@ AttributeSet attributes() {
 
    @Override
    public TableManipulationConfiguration create() {
-      return new TableManipulationConfiguration(attributes.protect());
+      return new TableManipulationConfiguration(attributes.protect(), batchSize);
```

Comment:
Sadly this didn't come out as well I would have hoped, but it seems a little better. Unfortunately due to the disconnectedness of the builders, there seems to be no way to pass the value back. This is fine though, we just need to make sure to remove this in 10.0 :)